projects/storefrontlib/src/layout/layout.module.ts

```diff
@@ -4,9 +4,15 @@ import { OutletRefModule } from '../cms-structure/outlet/outlet-ref/outlet-ref.m
 import { defaultLayoutConfig } from './config/default-layout.config';
 import { DirectionModule } from './direction/direction.module';
 import { LaunchDialogModule } from './launch-dialog/index';
+import { SiteThemeModule } from './theme/site-theme.module';
 
 @NgModule({
-  imports: [OutletRefModule, LaunchDialogModule.forRoot(), DirectionModule],
+  imports: [
+    OutletRefModule,
+    LaunchDialogModule.forRoot(),
+    DirectionModule,
+    SiteThemeModule,
```

Comment:
Are we still happy with this name? We also don't say SiteDirection. I can imagine we use just `ThemeModule` or `ThemingModule`.

Reply:
The same would apply to the service.

Reply:
I named it SiteTheme......... You're right we can drop `Site`.